crates/starfish/core/src/block_header.rs

```diff
@@ -877,14 +865,12 @@ impl TryFrom<SerializedHeaderAndTransactions> for VerifiedBlock {
         let transactions: Vec<Transaction> =
             bcs::from_bytes(&serialized_block.serialized_transactions)
                 .map_err(ConsensusError::MalformedTransactions)?;
-        // TODO: do we need to check the signature here?
```

Comment:
#8221 issue was created to disuses weather the signature and transaction commitment need to be checked at all.